src/components/common/Header.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import { Link, useLocation } from 'react-router-dom';
+
+import { menu, close } from 'assets';
+
+import routes from 'routes/routes';
+
+const Header = () => {
+  const Options = [
+    {
+      title: 'About',
+      path: routes.about.path
+    },
+    {
+      title: 'Contact',
+      path: routes.contact.path
+    }
+  ];
+
+  const [optionsOpen, setOptionsOpen] = useState(false);
+
+  const location = useLocation();
+
+  const onOptionsOpen = () => {
+    setOptionsOpen((prevState) => !prevState);
+  };
+
+  return (
+    <header className="headerWrapper">
+      <button type="button" onClick={onOptionsOpen}>
+        <img
+          src={optionsOpen ? close : menu}
+          alt={optionsOpen ? 'close' : 'menu'}
```

Comment:
not a priority for the scope of this PR 🙂  just to mention that in this context "closes menu" or "open menu" it's more accurate. In general (and thinking about accessibility) alternative texts should give information about the functionality or behavior of the elements where we are using them, in this case, to open/close the menu. More info about it is here: https://webaim.org/techniques/alttext/

Reply:
added ✅